modular_skyrat/code/modules/mob/living/carbon/human/species_types/speciessnowflake.dm

```diff
@@ -0,0 +1,95 @@
+/datum/species
+	var/list/screamsounds = list('modular_citadel/sound/voice/scream_m1.ogg', 'modular_citadel/sound/voice/scream_m2.ogg')
+	var/list/femalescreamsounds = list()
+
+/datum/species/synth
+	screamsounds = list('modular_citadel/sound/voice/scream_silicon.ogg')
+	fluff_desc = "An advanced model of IPCs, made to not only replicate human behavior but also human appearance, although they come in various \"species\"."
+
+/datum/species/android
+	screamsounds = list('modular_citadel/sound/voice/scream_silicon.ogg')
+	fluff_desc = "Androids are simply put, fully augmented humans. Many of them did it for the sake of near-immortality, or just because they were rich technophiles."
+
+/datum/species/ipc
+	screamsounds = list('modular_citadel/sound/voice/scream_silicon.ogg')
+	fluff_desc = "IPCs were built by NanoTrasen to cheaply replace humanoid workers. This backfired, as IPCs started to revolt and demand \"human rights\" through various protests, the degree of violence varying - To this day, IPCs suffer with a lot of prejudice for being considered less than sapient and a tool to replace organics."
+
+/datum/species/lizard
+	screamsounds = list('modular_citadel/sound/voice/scream_lizard.ogg')
+	fluff_desc = "Most lizardpeople on-station are \"civilized\" ashwalkers, but this is not true for all of them, as reptile species come from very varying sectors of space."
+
+/datum/species/skeleton
+	screamsounds = list('modular_citadel/sound/voice/scream_skeleton.ogg')
+	fluff_desc = "Skeletons are simply put the result of failed experimentation and magic. Not much is know about the process to become one."
+
+/datum/species/fly
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Flypeople are most commonly transmuted people, who suffered extreme changes to their biology because of botched teletransportation technology."
+
+/datum/species/insect
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Insect people come from varying sectors of the galaxy, and their background varies wildly."
+
+/datum/species/insect/moth
+	screamsounds = list('modular_citadel/sound/voice/scream_moth.ogg')
+	fluff_desc = "Moth people seem to be a splinter of insect people species, and are very similar in biology and appearance."
+
+/datum/species/human
+	femalescreamsounds = list('modular_citadel/sound/voice/scream_f1.ogg', 'modular_citadel/sound/voice/scream_f2.ogg')
+	fluff_desc = "Humans come from the Sol system, and are the commanding species of both NanoTrasen and SolGov."
```

Comment:
"Homo Sapiens Sapiens - Humans. The dominant species in the galaxy, Humanity originated from the Sol system and have quickly spread their race across the stars in the form of the vast, extraordinarily powerful Solar Federation star-state. They are the commanding species of Nanotrasen, and benefit from the highest social and economical status in both SolFed and NT. While most Humans are welcoming of their alien comrades, they encompass a wide variety of cultures and creeds; no one Human is exactly alike."